kalman.py

```diff
@@ -57,7 +57,7 @@ class Kalman:
   # How good our model is of the drone position.
   POSITION_MODEL_UNCERTAINTY = 0.001
   # How good our model is of the drone velocity.
-  VELOCITY_MODEL_UNCERTAINTY = 0.05
+  VELOCITY_MODEL_UNCERTAINTY = 0.0
```

Comment:
Well, in this case, since the model is just our velocity measurements, and thus, the "model" and the measurements match up exactly, the model is indeed perfect as far as it's concerned.